include/ebpf_api.h

```diff
@@ -564,6 +564,20 @@ extern "C"
     _Must_inspect_result_ ebpf_result_t
     ebpf_program_test_run(fd_t program_fd, _Inout_ ebpf_test_run_options_t* options) EBPF_NO_EXCEPT;
 
+    /**
+     * @brief Write data into the ring buffer map.
```

Comment:
```suggestion
     * @brief Write data into a ring buffer map.
```
Also I think "ring buffer map" should be expanded to say what type of ring buffer map, since this shouldn't be permitted in all ring buffer maps.

Reply:
Resolving this per meeting discussion, should use flags.